tests/catch/unit/stream/hipStreamWaitEvent.cc

```diff
@@ -59,6 +65,19 @@ TEST_CASE("Unit_hipStreamWaitEvent_Negative") {
   }
 }
 
+#if !HT_NVIDIA
```

Comment:
Please put a comment why this test is disabled on Nvidia. If there is JIRA defect created for that then mention that in the comment.

Reply:
I would like to understand why this happens and try to solve the issue instead of avoiding it on NV.

Reply:
This define has been set following the example of hipStreamQuery tests. The API is not supposed to cause Segmentation fault, as this is not the observed behavior when CUDA API is used. Defect is created to address the issue.